README.md

```diff
@@ -11,7 +11,7 @@ For full installable versions of the OpenBazaar app, with the server and client
 To create a local development copy of the reference client, clone the client repository into a directory of your choice:
 - `git clone https://github.com/OpenBazaar/openbazaar-desktop`
 
-Make sure you have the latest versions of Node.js and NPM installed.
+Make sure you have Node.js 8.9.2 and NPM 5.5.1 installed.
```

Comment:
shouldn't this have "or greater"?

Reply:
My thought is those are the versions I've tested up to, and even if it's not likely later versions would break anything, we can't be sure.

Reply:
hmm... I'm not sure I've ever seen anything like this (although I certainly haven't seen everything :)).

The thing is, for example, if the latest version of node becomes 8.9.9, I would think the user should just install latest and not go through the pain of having to install an old version.

Reply:
We could say make sure you have the latest versions of node 8 and NPM 5, and then say it's been tested on 8.9.2 and 5.5.1 specifically. 

Though we should make a habit of updating both of them and checking to make sure nothing breaks periodically.